src/libraries/System.Diagnostics.StackTrace/tests/StackTraceTests.cs

```diff
@@ -300,6 +303,66 @@ public void ToString_NullFrame_ThrowsNullReferenceException()
             Assert.Equal(Environment.NewLine, stackTrace.ToString());
         }
 
+        [ConditionalFact(typeof(RemoteExecutor), nameof(RemoteExecutor.IsSupported))]
+        public void ToString_ShowILOffset()
+        {
+            string SourceTestAssemblyPath = Path.Combine(Environment.CurrentDirectory, "ExceptionTestAssembly.dll");
+
+            // Normal loading case
+            RemoteExecutor.Invoke((asmPath) =>
+            {
+                AppContext.SetSwitch("Switch.System.Diagnostics.StackTrace.ShowILOffsets", true);
+                var asm = Assembly.LoadFrom(asmPath);
+                try
+                {
+                    asm.GetType("Program").GetMethod("Foo").Invoke(null, null);
+                }
+                catch (Exception e)
+                {
+                    Assert.Contains(":token ", e.InnerException.StackTrace);
```

Comment:
Can we make these asserts a little stronger? They don't need to have the exact token value but I would expect everything else ( type name, method name, assembly name, IL offset, whitespace, leading 0x06 on the token) is all predictable.

Reply:
I'll update